### PR TITLE
[Azure CDN] Cmdlets for Multi-Origin & Private Link Functionality 

### PR DESCRIPTION
**Pull request by jorinmejia**
## Description

This PR brings new and modified cmdlets for the Multi-Origin and Private Link functionality for the Azure CDN service.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/CDN/CDN/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

[Design Review](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/729)

### CODE REVIEW

**Comment by dingmeng-xue** [comment]
@jorinmejia , test case is failed. Please check the log 
https://dev.azure.com/azure-sdk/29ec6040-b234-4e31-b139-33dc4287b756/_apis/build/builds/568042/logs/33

File: src/Cdn/Cdn/Endpoint/NewAzureRmCdnEndpoint.cs [hunk 1]
```diff
@@ -103,6 +103,55 @@ public class NewAzureRmCdnEndpoint : AzureCdnCmdletBase
...
+        [Parameter(Mandatory = false, HelpMessage = "Azure CDN origin group ids.")]
+        [ValidateNotNullOrEmpty]
+        public List<string> OriginId { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "The name of the origin group.")]
+        public string OriginGroupName { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "The number of seconds between health probes.")]
+        [ValidateNotNullOrEmpty]
+        public int? OriginGroupProbeIntervalInSeconds { get; set; }
```
Comment (isra-fel) [inline src/Cdn/Cdn/Endpoint/NewAzureRmCdnEndpoint.cs:139]:
```suggestion
        public int OriginGroupProbeIntervalInSeconds { get; set; }
```
It's discuraged to type an *optional* parameter with a nullable type. Users may wonder if there's difference between not specifying it and specify a `$null`. If the SDK requires a nullable type, you need to do the conversion.

Please apply this to all the new parameters. Thanks

**Review by isra-fel — CHANGES_REQUESTED** [review]

**Comment by jorinmejia** [comment]
@dingmeng-xue / @isra-fel can I please get a follow up review for this PR?

File: src/Cdn/Cdn/ChangeLog.md [hunk 2]
```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 
+## Version 1.4.4
```
[the remark this answers is not in the document]
Comment (isra-fel) [reply src/Cdn/Cdn/ChangeLog.md]: What I meat is to remove the "## Version 1.4.4" line but keep your changelog line.
If you look at the first lines of changelog.md there's a great example:
```md
<!--
    Please leave this section at the top of the change log.

    Changes for the upcoming release should go under the section titled "Upcoming Release", and should adhere to the following format:

    ## Upcoming Release
    * Overview of change #1
        - Additional information about change #1
    * Overview of change #2
        - Additional information about change #2
        - Additional information about change #2
    * Overview of change #3
    * Overview of change #4
        - Additional information about change #4

    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
    * Overview of change #1
        - Additional information about change #1
-->
```

**Comment by isra-fel** [comment]
And please remove the temp SDK packages `tools/LocalFeed/Microsoft.Azure.Management.Cdn.4.3.0-preview.nupkg` and `tools/LocalNugetFeed`. I don't think you are using them anymore.

**Review by isra-fel — APPROVED** [review]

**Pull request merged**